.readthedocs.yaml

```
@@ -9,21 +9,25 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.9"
-    # You can also specify other tool versions:
-    # nodejs: "19"
-    # rust: "1.64"
-    # golang: "1.19"
+    python: mambaforge-4.10
+
+conda:
+  environment: docs/rtd_environment.yaml
 
 # Build documentation in the docs/ directory with Sphinx
 sphinx:
-   configuration: docs/conf.py
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: true
```

sorry! I shouldn't have set this to `true`; it's causing the RTD build to fail
```suggestion
  fail_on_warning: false
```